targetMapper.py

```diff
@@ -0,0 +1,71 @@
+"""
+PSU UAS ...
+Author: Ted Tasman
+Date: 2024-10-17
+Description:    This module converts craft GPS coordinates and heading, 
+                along with a geosensor displacement into a target GPS coordinate
+Version: v1.0.0
+"""
+
+from math import sin, cos, pi
+import geosensor
+from typing import List
+
+EARTH_RADIUS = 6371000  # Earth's radius in meters
+
+class Craft:
+
+    def __init__(self, lat: float=0, lon: float=0, alt: float=0, heading: float=0):
+        self.lat = lat
+        self.lon = lon
+        self.alt = alt
+        self.heading = heading
```

Comment:
My want to consider roll as well. I assume you've already accounted for the combined pitch of the camera and the craft?